config.js

```diff
@@ -22,8 +22,12 @@ const protocol          = pe.PLUGIN_PROTOCOL || pe.SNOW_PROTOCOL || 'https';
 const snowProdInstance  = pe.PLUGIN_PROD_HOST || pe.SNOW_PROD_HOST || 'lssiprod.service-now.com';
 const snowTestInstance  = pe.PLUGIN_TEST_HOST || pe.SNOW_TEST_HOST || 'lssitest.service-now.com';
 const sendToProd        = /^true$/i.test(pe.PLUGIN_SEND_TO_PROD) || /^prod/i.test(pe.DEPLOY_TO || pe.SNOW_DEPLOY_TO);
-const username          = pe.PLUGIN_USERNAME || pe.SNOW_USER;
-const password          = pe.PLUGIN_PASSWORD || pe.SNOW_PASS;
+const testUser          = pe.PLUGIN_TEST_USER || pe.SNOW_TEST_USER;
+const prodUser          = pe.PLUGIN_PROD_USER || pe.SNOW_PROD_USER;
+const username          = pe.PLUGIN_USERNAME || (sendToProd ? prodUser : testUser) || pe.SNOW_USER;
```

Comment:
This is a little strange. It seems to me that it would be less confusing if `pe.SNOW_USER` took precedence as elsewhere the `PLUGIN_` vars are generally equivalent to `SNOW_` ones.